cmd/skywire-cli/commands/node/gen-config.go

```diff
@@ -149,6 +149,14 @@ func defaultConfig() *visor.Config {
 
 	conf.AppServerSockFile = "/tmp/visor_" + pk.Hex() + ".sock"
 
+	if conf.Retrier == nil {
+		conf.Retrier = &visor.RetrierConfig{
```

Comment:
I would create a new variable or function for default config to be able to reuse it. `3-5-2` values are used many times.